src/notes/state/create-note.ts

```diff
@@ -1,6 +1,5 @@
-export default function createNote(rawName: string): void {
-  const name = rawName.trim();
-  if (!name) {
+export default function createNote(newNoteName: string): void {
+  if (!newNoteName) {
```

Comment:
New note name comes in already trimmed, as it is now a `Modal.tsx` responsibility to provide a clean, trimmed input. Create function now does exactly what it is told to do. There are no further changes to the given name.